src/components/DropdownList.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+
+import { FC } from 'react';
+import { Person } from '../types/Person';
+
+interface Props {
+  people: Person[];
+  selectPerson: (person: Person) => void;
+}
+
+export const Dropdown: FC<Props> = ({
+  selectPerson,
+  people,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {people.length !== 0
```

Comment:
probably you meant this one?
```suggestion
        {people.length > 0
```